nats/pkg/config/defaultconfig.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+// NOTE: Most of this configuration is not needed to keep it as simple as possible
+// TODO: Clean up unneeded configuration
+
+func DefaultConfig() *Config {
+	return &Config{
+		Service: Service{
+			Name: "nats",
+		},
+		Nats: Nats{
+			Host: "127.0.0.1",
+			Port: 4222,
```

Comment:
@kobergj I know this is the NATS default port. But it seems we have a policy to only listen on a specific port range. (See https://github.com/owncloud/ocis/wiki/Official-Port-Ranges). Yeah, I also know I am a bit late to the party for this PR but I also just learned about that document ;-)